en/docs/develop/product-apis/gateway-apis/gateway-v1/gateway-v1.yaml

```diff
@@ -0,0 +1,636 @@
+    post:
+      security:
+        - OAuth2Security:
+            - apim:api_publish
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -X POST \"https://127.0.0.1:9443/api/am/gateway/v1/deploy-api?apiId=890a4f4d-09eb-4877-a323-57f6ce2ed79b&apiName=PizzaAPI&label=Food\""
+      x-wso2-request: |
+        POST https://127.0.0.1:9443/api/am/gateway/v1/undeploy-api?apiId=890a4f4d-09eb-4877-a323-57f6ce2ed79b&apiName=PizzaAPI&label=Food
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+      x-wso2-response: "HTTP/1.1 200 OK"
+      summary: Uneploy the API in the gateway
+      description: |
+        This operation is used to undeploy an API in the gateway.
+      parameters:
+        - name: apiName
+          description: |
+            Name of the API
+          in: query
+          type: string
+          required: true
+        - name: label
+          description: |
+            Label of the API Gateway
+          in: query
+          type: string
+          required: true
+        - $ref: '#/parameters/apiId-Q'
+
+      tags:
+        - Undeploy API
+      responses:
+        200:
+          description: |
+            OK.
+            API successfully undeployed from the Gateway.
+          schema:
+            $ref: '#/definitions/DeployResponse'
+        202:
+          description: |
+            Accepted.
+            The request has been accepted.
+          schema:
+            $ref: '#/definitions/DeployResponse'
+        400:
+          description: |
+            Bad Request.
+            Invalid request or validation error
+          schema:
+            $ref: '#/definitions/Error'
+        404:
+          description: |
+            Not Found.
+            Requested API does not exist.
+          schema:
+            $ref: '#/definitions/Error'
+        412:
+          description: |
+            Precondition Failed.
+            The request has not been performed because one of the preconditions is not met.
+          schema:
+            $ref: '#/definitions/Error'
+
+  /api-artifact:
+
+    #-----------------------------------------------------
+    # Get API Artifact
+    #-----------------------------------------------------
+    get:
+      security:
+        - OAuth2Security:
+            - apim:api_publish
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" -X GET \"https://127.0.0.1:9443/api/am/gateway/v1/api-artifact?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com\""
+      x-wso2-request: |
+        GET https://127.0.0.1:9443/api/am/gateway/v1/api-artifact?apiName=PizzaAPI&version=1.0&tenantDomain=registry.com
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+      x-wso2-response: "HTTP/1.1 200 OK"
+      summary: Get API artifact from the storage
+      description: |
+        This operation is used to get the API artifact from the storage
```

Comment:
better to have a meaningful description with all the details returning

Reply:
https://github.com/wso2/docs-apim/pull/1527/commits/7fb0b6cb28a8074e263fe30b1f22a77a2a662d15